### DISCUSSION TITLE
Adding feedback in document approval elsa workflow

### DISCUSSION THREAD

**Discussion by MohammedNash** [elsa-workflows/elsa-core#2540] [Q&A]
In document workflow, after I reject/approve the document in the mail, I want it to take to the browser where I can add a comment in the comment box and then pass that comment back to the employee through email. How can it be done in UI workflow designer? - 
I have tried adding comments in console and pass it through mail.
@sfmskywalker

**Comment by sfmskywalker** [comment]
This [SO answer](https://stackoverflow.com/a/70070474/690374) offers a few pointers that might help get you started.
